ts/packages/ts/src/commands/ExportCommand.ts

```diff
@@ -37,10 +38,23 @@ export class ExportCommand extends Command<ExportCommandReturn> {
     });
   }
 
+  private filterPipe(pipe: EvalCommandReturn) {
+    const { filter } = this.parameters;
+    if (!filter) {
+      return _.pickBy(pipe, ({ context }) => !context.cfgu.hidden);
+    }
+    return _.pickBy(pipe, filter);
+  }
```

Comment:
This logic feels inaccurate, if a filter is provided then the `hidden` filter is skipped and causes all hidden keys to be shown. I think the hidden filter should always happen on top of the provided filter.